assets/app/scripts/services/data.js

```diff
@@ -641,6 +640,9 @@ angular.module('openshiftConsole')
         return;
       }
 
+      // Reset the retries count once we've received a message with no error.
+      this._watchWebsocketRetries(type, context, 0);
```

Comment:
put it after the _resourceVersion/_data calls... if there's an error there, that could result in looping